src/_playground/Routes.js

```diff
@@ -345,7 +371,14 @@ export class Routes extends Component {
                             {navItems}
                         </nav>
                     </div>
-                    <div className='frDocs-Content'>
+                    <div className={docsContentClasses}>
+                        <Button
+                            aria-expanded={this.state.showSideNav}
+                            aria-label='Toggle Navigation'
+                            className='fd-button sap-icon--menu2 fd-button--standard fd-button--light sidebar-toggle'
```

Comment:
We can remove all the classNames that aren't unique to this Button (ie keep sidebar-toggle). The component should handle adding all the other classes for you.